src/main/pages/setup-kubernetes/kubernetes-config.adoc

```diff
@@ -1,41 +1,35 @@
 ---
-title: "Configuration: Kuberentes"
+title: "Configuring Kubernetes"
 keywords: kubernetes, configuration
 tags: [installation, kubernetes]
 sidebar: che_6_docs
 permalink: kubernetes-config.html
 folder: setup-kubernetes
 ---
 
-[id="how-it-works"]
-== How It Works
+You can configure the behavior of the Che server by passing environment variables to the Che deployment.
 
-Che server behavior can be configured by passing environment variables to Che deployment.
+There are multiple ways to edit the Che deployment to add new or edit existing environment variables:
 
-There are multiple ways to edit Che deployment to add new or edit existing envs:
+* To open the Che deployment YAML file in the nano editor, use the `kubectl edit dc/che` command. The Vim editor is used by default.
```

Comment:
Doesn't make much sense -- is it nano or Vim? :) Not to mention that talking about the editor is superfluous.
```suggestion
* To open the Che deployment YAML file in the configured editor, use the `kubectl edit dc/che` command.
```